setup.cfg

```diff
@@ -103,6 +103,7 @@ filterwarnings =
     ignore:.*np\.int0.*is a deprecated alias for:DeprecationWarning
     ignore:zmq\.eventloop\.ioloop is deprecated in pyzmq:DeprecationWarning
     ignore:AsdfInFits has been deprecated
+    ignore:specutils uses the deprecated entry point asdf_extensions
```

Comment:
This should not have been here. How did this sneak in?

Reply:
I have absolutely no idea 🤔

Reply:
Got rid of it with a rebase 🤷